website/docs/r/dx_macsec_key.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Direct Connect"
+layout: "aws"
+page_title: "AWS: aws_dx_macsec_key"
+description: |-
+  Provides a MAC Security (MACSec) secret key resource for use with Direct Connect.
+---
+
+# Resource: aws_dx_macsec_key
```

Comment:
Have you considered naming this resource `aws_dx_macsec_key_association`? Since MACsec keys can only be associated/described/dissociated, renaming the resource may help preventing any confusion. 
```suggestion
# Resource: aws_dx_macsec_key_association
```